apps/native-component-list/src/screens/FileSystemScreen.tsx

```diff
@@ -233,6 +233,23 @@ export default class FileSystemScreen extends React.Component<object, State> {
     }
   };
 
+  _createUTF8Uri = async () => {
```

Comment:
Add a separate test case for a path like: `/hello%2Fworld.txt`. I believe it should not create `/hello/world.txt` (specifically, no directory named "hello").

Reply:
Added a couple more cases to the units tests covering this